### DISCUSSION TITLE
Welcome to Eloquent Discussions!

### DISCUSSION THREAD

**Discussion by mdbergmann** [mdbergmann/Eloquent#68] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Thomas-Cee** [comment]
Hey! I use Eloquent DAILY. I pastor a little church in Concord California, and I just like things that work. I don't like extra "stuff". I ran across Eloquent years ago and have been using it ever since. I wanted to give a sincere thank you to all the work you have put into this open source project. I use it so often, I was thinking the other day, "does the author have any idea how much his work is appreciated??"
Thank you deeply! What a great program you've built!
-Thomas

**Reply by mdbergmann** [reply]
Thanks, Thomas.
It's much appreciated.
